### DISCUSSION TITLE
A Cross-Platform GUI Application

### DISCUSSION THREAD

**Discussion by SecularSteve** [WeblateOrg/weblate#10888] [Ideas]
I'm curious to know if people are interested in seeing a Weblate application that can be used to translate other applications easier on devices where accessing it by web isn't a picnic.

Mobile and Tablet devices for example.
Not everyone has access to a keyboard 24/7. And having a more streamlined solution of using it on certain devices seems handy.

Kind of like what Poedit is to CrowdIn if it was less PC focused.

But that's just my opinion.

**Comment by nijel** [comment]
You can download files for offline translation and use any tool you like (including Poedit), see https://docs.weblate.org/en/latest/user/files.html.
